readme.txt

```diff
@@ -3,6 +3,8 @@
 = [TBD] TBD =
 
 * Tweak - Rename the `Controller_Test_Case` `setUp` and `tearDown` methods and annotate them with `@before` and `@after` annotations to improve PHPUnit version cross-compat.
+* Fix - Add the dir and filename of `event-automator` in the Plugins_API so that the upsell CTA button text and links are shown properly in the Help section of TEC.
+* Feat - Add Events Schedule Manager cards in the Help and App Shop admin pages to promote.
```

Comment:
Add the [ticket ID] at the end of the changelog if there is a ticket associated.